src/module.ts

```diff
@@ -28,15 +27,16 @@ export class SsvUxModule {
 					useFactory: _moduleConfigFactory,
 					deps: [_MODULE_CONFIG],
 				},
-				{ provide: _MODULE_CONFIG, useValue: config || UX_DEFAULT_CONFIG },
+				{ provide: _MODULE_CONFIG, useValue: config },
 			],
 		};
 	}
 }
 
 /** @internal */
-export function _moduleConfigFactory(config: UxOptions | (() => UxOptions)) {
-	return typeof config === "function" ? config() : config;
+export function _moduleConfigFactory(config: UxOptions | (() => UxOptions)): UxOptions {
+	const uxOptions = typeof config === "function" ? config() : config;
+	return _.merge(UX_DEFAULT_CONFIG, uxOptions || UX_DEFAULT_CONFIG);
 }
```

Comment:
merge the provided options with the default options.

the fallback to default config can be adjusted to empty object instead - `uxOptions || UX_DEFAULT_CONFIG`